docs/modules/ROOT/pages/ios_task_scheduling.adoc

```diff
@@ -0,0 +1,33 @@
+= Task Scheduling
+
+Background tasks are scheduled based on the app's current context, and that context is based on a combination of factors. 
+These factors include:
+
+* Is the app backgrounded or in the foreground?
+* Is WiFi available?
+* Is the device in Low Power Mode?
+* Is the device connected to external power?
+* Has the photo library has changed?
+
+== Tasks 
+
+Currently, two tasks are available:
+
+* *Instant photo uploads:* This task is triggered when:
+** The photo library has new assets
```

Comment:
```suggestion
** The photo library has new photos, and;
```
IMO "ordinary mortals" will not know that "assets" means photos.
IMO this task is triggered when A and B are both true - so we can be explicit about that